### PR TITLE
Issue 449: security evaluation process - documentation update

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
This issue is to update the existing documentation to reflect a new security evaluation process

### CODE REVIEW

**Review by JustinCappos — APPROVED** [review]
I've read this over and the changes all seem fine to me.  I'm missing some context for a few of them, but they are all logical decisions.  I imagine we will need further refinement, but this feels like an important step forward!

File: .github/ISSUE_TEMPLATE/joint-evaluation.md
Comment (pragashj) [inline .github/ISSUE_TEMPLATE/joint-evaluation.md:5]: Do we still want to label them as assessment?

Comment (TheFoxAtWork) [reply .github/ISSUE_TEMPLATE/joint-evaluation.md:5]: @pragashj until #479 and #488 are merged in, it should still be called 'assessment'

File: assessments/README.md
Comment (lumjjb) [reply assessments/README.md:92]: looks like this is mentioned later on

File: assessments/guide/README.md
Comment (lumjjb) [inline assessments/guide/README.md:80]: can we link to security evaluation facilitator

Comment (lumjjb) [reply assessments/guide/README.md:80]: I think we may have to update this in the README.md and governance.md as well

File: assessments/guide/README.md
Comment (lumjjb) [inline assessments/guide/README.md:203]: nit: `/assessments/projects/project-name` folder

File: assessments/guide/security-reviewer.md
Comment (lumjjb) [inline assessments/guide/security-reviewer.md:53]: link should point to the .md?

**Review by lumjjb — COMMENTED** [review]
Awesome work here! Have a couple nits and requests, if not LGTMed! This is great, love the revamp.

File: assessments/guide/README.md
Comment (TheFoxAtWork) [reply assessments/guide/README.md]: @lumjjb which update?  content changes or the link to the security evaluation facillitator role?

**Comment by TheFoxAtWork** [comment]
@lumjjb made the updates u requested, question on one of them

**Comment by TheFoxAtWork** [comment]
change evaluation to review.

**Comment by lumjjb** [comment]
tag #449 for ref

File: .github/ISSUE_TEMPLATE/joint-evaluation.md
Comment (lumjjb) [inline .github/ISSUE_TEMPLATE/joint-evaluation.md:3]:
```suggestion
about: To request a joint evaluation or track progress on an active evaluation
```
nit

File: assessments/README.md
Comment (lumjjb) [inline assessments/README.md:1]: is there a reason we distinguish between review and evaluation? Feel like these should be consistent

File: assessments/guide/security-reviewer.md
Comment (lumjjb) [inline assessments/guide/security-reviewer.md:181]:
```suggestion
a SIG-Security chair must approve the security review team. Reasons for
```

**Review by lumjjb — COMMENTED** [review]
Very minor nits. But I find the use of "review" and "evaluation" a bit confusing. A first time reader may confuse them for 2 different things. I think both terms are fine, but we should stick to one, in the evaluation case, we probably need to update the wording of "security evaluator"

**Review by lumjjb — APPROVED** [review]
LGTM!

File: assessments/guide/security-reviewer.md
Comment (ultrasaurus) [inline assessments/guide/security-reviewer.md:160]: the team will not typically be conflict-free, so this is confusing, I propose "Asserting team readiness to conduct a balanced review" (I'll include in my PR)

**Review by ultrasaurus — CHANGES_REQUESTED** [review]
I fixed the small wording suggestions here: https://github.com/TheFoxAtWork/sig-security/pull/1

I am requesting a bigger change, which is to separate the CNCF stages into a separate PR (and get this one submitted first after our internal SIG-Security review), so TOC review can focus on smaller PR

**Comment by ultrasaurus** [comment]
thanks for quick response @TheFoxAtWork !

one remaining open question (from above):  "Is this [joint review] a separate document now? previously I think we made suggested edits in the self-eval and the rest of the content went in the README"

File: assessments/guide/README.md
Comment (TheFoxAtWork) [inline assessments/guide/README.md:192]:
```suggestion
   * **Lead security reviewer** places the [summary](joint-readme-template.md) into branch for finalization
```

**Review by TheFoxAtWork — COMMENTED** [review]
> thanks for quick response @TheFoxAtWork !
> 
> one remaining open question (from above): "Is this [joint review] a separate document now? previously I think we made suggested edits in the self-eval and the rest of the content went in the README"

Below is the list of files/changes and brief summary of where they fit/what they are, @ultrasaurus the self-assessment, joint-review, and joint readme are all separate in the repo.  The self-assessment is significantly smaller than the joint-review and intended to be a starting point for less mature projects.  The joint review pulls a lot of content from the self-assessment, with an expectation those are updated as appropriate and expanded during the joint-review.  The joint-readme is the template to extract the relevant, summarized content from the joint-review for the project's folder in the repo.

* Github issue template update - name change, content update
* assessments/README.md - overview of the security review process
* assessments/guide/README.md - detailed description of the security review process, steps, document links, templates, etc.
* assessments/guide/joint-readme-template.md - this is the final summary written by the review team that serves as the Project's README.md in the project folder when the security review's joint-review is complete.
* assessments/guide/joint-review.md - template/outline for the joint-review document that builds on top of the self-assessment.  May be used independently from the self-assessment for more mature projects
* assessments/guide/project-lead.md - updates to the content and naming
* assessments/guide/review-survey.md - simple survery to ascertain effectiveness and experience of the security review process
* assessments/guide/security-reviewer.md - updates to the content and naming
* assessments/guide/self-assessment.md - template/outline for the self-assessment used to later build the joint-review.  Recommended for  less mature projects looking to initiate security thinking and documentation.

File: assessments/README.md
Comment (TheFoxAtWork) [reply assessments/README.md:80]: https://github.com/cncf/sig-security/pull/488#pullrequestreview-588680948

**Comment by ultrasaurus** [comment]
thanks @TheFoxAtWork for the summary!

The change of separating self-assessment and joint-review documents seems like it will reduce initial effort of project lead and put more work onto the security review team.  This seems important since the new process allows for self-assessment and joint-review to be spaced farther apart in time than has been the case in the past.  As a past reviewer, I'd be fine with this change if I were to lead a future review.

I think this is on the agenda for the meeting tomorrow, so ok to discuss then.  It would be great to capture motivation and check in with past project leads & other reviewers (if you haven't done so already) to validate that they believe this change is an improvement :)

**Comment by lumjjb** [comment]
@ultrasaurus looks like there were changes made based on your comments, can you verify that your comments have been resolved? I think with that, we should be good to merge soon!

**Review by ultrasaurus — APPROVED** [review]
LGTM!

**Pull request merged**
